devlib/target.py

```diff
@@ -2892,6 +2895,99 @@ def _resolve_paths(self):
         if self.executables_directory is None:
             self.executables_directory = '/tmp/devlib-target/bin'
 
+class QEMULinuxTarget(LinuxTarget):
```

Comment:
I don't think it's really an issue but I was wondering if we really want to make that QEMU runner a subclass of LinuxTarget. If we do that, we can easily end up with a combinatorial explosion of the number of subclasses with just a few "modifier" classes like that, and the naming nightmare that goes with it.

Maybe a better approach to the problem is to make that QEMU thingy a separate class (e.g. let's say a TargetRunner class with a QEMUTargetRunner subclass), and then just pass the target class to use as a parameter. The created target can be accessed as a `.target` attribute. This way we also avoid any possible clash between methods defined here and current + future methods of Target classes.